### PR TITLE
Change Survey Questions to NonSwipableViewPager

### LINKED ISSUE #271: Background questions should be in a view pager
- Background question animation freeze #264

### PR DESCRIPTION
**Pull request by alextac98**
Fixes #271 
Restyles drop-down menu to fit all test sizes

Known Bugs:

- [x] Review Page doesn't display any information

### CODE REVIEW

**Comment by bhylak** [comment]
"Review Page doesn't display any information" 👎

File: app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/SurveyQuestionsFrag.java [hunk 1]
```diff
@@ -65,8 +65,8 @@ public void setAnswerRequired(boolean answerRequired) {
     public void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
-        mQuestionAdapter = new SurveyQuestionAdapter(this, getFragmentManager(), mSurveyReviewAdapter);
         mSurveyReviewAdapter = new SurveyQuestionReviewAdapter();
+        mQuestionAdapter = new SurveyQuestionAdapter(this, getFragmentManager(), mSurveyReviewAdapter);
```
Comment (bhylak) [inline app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/SurveyQuestionsFrag.java]: @alextac98 this is the change that fixed it all :)

**Comment by alextac98** [comment]
@bhylak fixed the problem

**Comment by alextac98** [comment]
@bhylak codacy issues are correct...

**Pull request merged**